Userland/Libraries/LibWeb/WebDriver/Capabilities.cpp

```diff
@@ -52,7 +52,7 @@ static Response deserialize_as_ladybird_options(JsonValue value)
 
     auto const& object = value.as_object();
 
-    if (auto headless = object.get_bool("headless"sv); headless.has_value())
+    if (auto headless = object.get("headless"sv); headless.has_value() && !headless->is_bool())
```

Comment:
What's the difference? `get_bool()` does the exact same thing, AFAICT:

```cpp
Optional<bool> JsonObject::get_bool(StringView key) const
{
    auto maybe_value = get(key);
    if (maybe_value.has_value() && maybe_value->is_bool())
        return maybe_value->as_bool();
    return {};
}
```

Reply:
The intent here is to checking if you've specified `headless` but not as a boolean.

The way it's written right now, we'll reject the payload when `headless` *is* a boolean.

Reply:
Oh, I see. Keep it that way and add a `!` before `headless.has_value()` then?

Reply:
...unless it's optional, in which case nevermind :)

Reply:
That would fail if you've added the `serenity:ladybird` but didn't also add `headless`, which I didn't want to do when I wrote this originally

Reply:
:ninja: